Jellyfin.Server.Implementations/UserData/UserDataManager.cs

```diff
@@ -104,9 +104,9 @@ private async Task<UserItemData> GetUserDataAsync(User? user, BaseItem item)
     {
         ArgumentNullException.ThrowIfNull(user);
         ArgumentNullException.ThrowIfNull(item);
-        var dbContext = await _provider.CreateDbContextAsync().ConfigureAwait(false);
+        var dbContext = await _provider.CreateDbContextAsync().ConfigureAwait(true);
```

Comment:
No, you almost never want to set it to true. And it's a no-op anyway. We are not hosting Jellyfin anywhere that has a synchronization context

Reply:
Fixed